commit 1380952bd517e83ef2bf5305f26e55a4bfbc57fb
Author: Volker Krause <vkrause@kde.org>
Date:   Tue May 6 17:49:58 2025 +0200

    Don't propagate breadcrumb toolbar style to layers
    
    Since 2a6bd9e99936ec4 breadcrumb toolbars don't work on layers anymore,
    and arguably they don't make sense there.
    
    Testcase: Itinerary's about page in mobile mode. Becomes even more
    prominent with safe area margins applied.

diff --git a/src/controls/PageRow.qml b/src/controls/PageRow.qml
index e614a807..99e8883c 100644
--- a/src/controls/PageRow.qml
+++ b/src/controls/PageRow.qml
@@ -306,7 +306,9 @@ QT.Control {
                 dialog.open();
             } else {
                 // open as a layer
-                properties.globalToolBarStyle = root.globalToolBar.style
+                if (root.globalToolBar.style !== Kirigami.ApplicationHeaderStyle.Breadcrumb) {
+                    properties.globalToolBarStyle = root.globalToolBar.style
+                }
                 item = layers.push(page, properties);
                 Object.defineProperty(item, 'closeDialog', {
                     value: function() {
